CHANGELOG.md

```diff
@@ -6,26 +6,89 @@ v0.26 + 1
 * Improved `p_unlink` in `posix_w32.c` to try and make a file writable
   before sleeping in the retry loop to prevent unnecessary calls to sleep.
 
+* The CMake build infrastructure has been improved to speed up building time.
+
+* A new CMake option "-DUSE_HTTPS=<backend>" makes it possible to explicitly
+  choose an HTTP backend.
+
+* A new CMake option "-DSHA1_BACKEND=<backend>" makes it possible to explicitly
+  choose an SHA1 backend. The collision-detecting backend is now the default.
+
+* A new CMake option "-DUSE_BUNDLED_ZLIB" makes it possible to explicitly use
+  the bundled zlib library.
+
+* A new CMake option "-DENABLE_REPRODUCIBLE_BUILDS" makes it possible to
+  generate a reproducible static archive. This requires support from your
+  toolchain.
+
+* The minimum required CMake version has been bumped to 2.8.11.
+
 * Writing to a configuration file now preserves the case of the key given by the
   caller for the case-insensitive portions of the key (existing sections are
   used even if they don't match).
 
 * We now support conditional includes in configuration files.
 
+* Fix for handling re-reading of configuration files with includes.
+
+* Fix for reading patches which contain exact renames only.
+
+* Fix for reading patches with whitespace in the compared files' paths.
+
+* Fix for writing only the last refspec into FETCH_HEAD in case multiple
+  refspecs are being fetched.
```

Comment:
This makes it sound like the fix we did is that only the last refspec is written, when it's the other way around.  Maybe something like

> We now fill `FETCH_HEAD` from all passed refspecs instead of overwriting with the last one.